ReactNative/ReactTwitter/core/service/twitter-requests-service.js

```diff
@@ -0,0 +1,51 @@
+var OauthHelper = require('../service/oauth-helper.js')
+
+const CONSUMER_KEY = 'aqPSTs1FT2ndP7qXi247BtbXd'
+const CONSUMER_SECRET = 'pTvKlESMnsmc7DqP70CfmJypGgk9oaHPxgC7DkUVbxVZXtIGp6'
+
+const BASE_URL = 'https://api.twitter.com'
+const REQUEST_TOKEN = '/oauth/request_token'
+const CALLBACK_URL = 'react-twitter-oauth://callback'
+
+class TwitterRequestsService {
+
+  requestToken () {
+    let ouathHelper = new OauthHelper(CONSUMER_SECRET)
+    let time = new Date().getTime() / 1000 | 0
+    let nonce = OauthHelper.generateNonce()
+    let url = BASE_URL + REQUEST_TOKEN
+    const params = {
+      'oauth_callback': CALLBACK_URL,
+      'oauth_consumer_key': CONSUMER_KEY,
+      'oauth_nonce': nonce,
+      'oauth_signature_method': 'HMAC-SHA1',
+      'oauth_timestamp': time,
+      'oauth_version': '1.0'
+    }
+
+    return fetch(url, // eslint-disable-line no-undef
```

Comment:
You shouldn't use globals, as they are unmockable. Wrap it in your own class if you need to.